app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -304,4 +302,6 @@ class ApplicationModule(context: Context)
       s"Started Frontend $appName in mode ${environment.mode} at port ${application.configuration.getOptional[String]("http.port")}"
     )
   }
+
+  override def httpFilters: Seq[EssentialFilter] = frontendFiltersModule.httpFilters
```

Comment:
I'm juts curious... why this has been moved and changed to `def` ?

Reply:
It's defined as `def` inside `BuiltInComponents`. My IDE converts it as `def`. 
`  def httpFilters: Seq[EssentialFilter]`